### ISSUE TITLE
Verbiage for Inclusion and Safety

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/code-questions-bot#11]
I have these questions set up for the bot to ask, but I've commented most out. I don't want them to inadvertently hurt people. If you fit into these groups of people, please help.

Goals:
- As inclusive as possible
- Avoid hurting people
- Avoid outing people
- Avoid other issues

Need help with:
- as inclusive as possbile
- wording
- appropriateness

```python
    'What badass ladies are you following that everyone else should follow? \nWhy? \nTag \'em',
    # 'What badass dudes are you following that everyone else should follow? \nWhy? \nTag \'em',
    # 'What badass trans* peeps are you following that everyone else should follow? \nWhy? \nTag \'em \nnote: do not out someone who isn't out',
    # 'What badass POC are you following that everyone else should follow? \nWhy? \nTag \'em',
    # 'What badass underrepresented peeps are you following that everyone else should follow? \nWhy? \nTag \'em',
```

_If you'd like to help but prefer to be more private, DM on Twitter [Vicki_Langer](twtter.com/vicki_langer) or email me at vicki_langer@hotmail.com_

**Comment by achasveachas** [comment]
I'm pretty sure the last one should be fine.

The second and third to last one I'll wait for POC and/or trans folks to weigh in.

As for the first commented out one I see why that might be seen as reinforcing tech stereotypes. Maybe substitute "dudes" for "allies"?

**Comment by VickiLanger** [comment]
Thanks for your input!

> The second and third to last one I'll wait for POC and/or trans folks to weigh in.

Yeah, I’m hoping someone will assist and weigh in with their ideas.

> As for the first commented out one I see why that might be seen as reinforcing tech stereotypes. Maybe substitute "dudes" for "allies"?

I can see that. I do like that idea.

**Comment by achasveachas** [comment]
Perhaps changing "trans" to "queer" would make it sound more inclusive and less like it's specifically "targeting" trans folks?

(With the added benefit that you get suggestions for more awesome queer folks 😊)

**Comment by VickiLanger** [comment]
> Perhaps changing "trans" to "queer" would make it sound more inclusive and less like it's specifically "targeting" trans folks?
> 
> (With the added benefit that you get suggestions for more awesome queer folks 😊)

Yes! Thank you for that. I was trying to figure out a way to be non-target-y, yet inclusive. I like that it can now be more inclusive than it was. Also, I wouldn't want trans* people to feel like they couldn't be included in other questions. 

Here's the adjusted questions:
```python
'What badass ladies are you following that everyone else should follow? \nWhy? \nTag \'em',  # includes anyone cis, trans*, femme presenting, not femme presenting
'What badass queer/LGBTQIA peeps are you following that everyone else should follow? \nWhy? \nTag \'em \nnote: do not out someone who isn't out',
'What badass LGBTQIA allies are you following that everyone else should follow? \nWhy? \nTag \'em',
# 'What badass POC are you following that everyone else should follow? \nWhy? \nTag \'em',
# 'What badass underrepresented peeps are you following that everyone else should follow? \nWhy? \nTag \'em',
```

**Comment by am1983** [comment]
I definitely think the bot should ask folks about people of color they follow - but I would caution against using POC as a blanket statement for non-white folks. How about being more specific? 

"What bad ass Black folks are you following that everyone else should follow? Why? Tag 'em." 
"What bad ass Latinx folks are you following that everyone else should follow? Why? Tag 'em."

(This list is not all-inclusive and is just an example of how you might handle the situation.)

Using a blanket "POC" *could* be seen as a form of erasure. This is just my two cents on the subject. I welcome correction if my approach is problematic.

**Comment by Sharondio** [comment]
-ladies +women

**Comment by VickiLanger** [comment]
> I definitely think the bot should ask folks about people of color they follow - but I would caution against using POC as a blanket statement for non-white folks. How about being more specific?
> 
> "What bad ass Black folks are you following that everyone else should follow? Why? Tag 'em."
> "What bad ass Latinx folks are you following that everyone else should follow? Why? Tag 'em."
> 
> (This list is not all-inclusive and is just an example of how you might handle the situation.)
> 
> Using a blanket "POC" _could_ be seen as a form of erasure. This is just my two cents on the subject. I welcome correction if my approach is problematic.

Thank you, I like your idea to separate and give more exposure to more different groups. I completely agree. I felt like it was making it too generic and didn't like it.

Though, I am worried I would miss a group and I don't want to do that.

**Comment by VickiLanger** [comment]
> -ladies +women

Done. I made the change. May I ask why I should use "women" over "ladies"?

**Comment by Sharondio** [comment]
"Ladies" is a designation based largely on cultural norms. Women is what we are.

**Comment by VickiLanger** [comment]
> "Ladies" is a designation based largely on cultural norms. Women is what we are.

Thanks. Very good point. When writing it I hadn't thought of it that way.
